wiki/Modding_Guides/How_To_Get_Your_Map_Ranked/zh.md

```diff
@@ -1,24 +1,27 @@
-　　始终无法让你的[图被](ZH:Glossary#Beatmap "wikilink") Rank 吗？你来对地方了。
+Rank 图教战手册
+================
+
+始终无法让你的[图被](/wiki/Glossary) Rank 吗？你来对地方了。
 
 第 1 步：摸图 Modding
 ---------------------
 
-　　让自产图被 [Rank](ZH:Ranked "wikilink") 的第一步，也是最重要的一步，便是找人为你检查一下图。这个过程称为 [**Mod**](ZH:Modding "wikilink")，中文 osu! 圈中亦有人将其叫做**摸图**。你可以在 \#modreqs 频道、Working In Progress/Help（制作中的图／作图杂症板块）底下的子板块 Modding Queues（摸图串），或是在确认好对方的签名档、用户页面中所写的摸图规则之后直接询问对方等方式来找人帮你看看图。记住：所有 Mod 都有其用处，因为任何来自其他人的摸后感与想法都能为你提供更多观点，让你能将图做得更好。请务必考虑过任何来自于摸图者的建议，若你否决了其中一项，别忘了在图的讨论串底下发一篇详尽的回应清楚说明原因。这会让他们的摸图技巧进步，同时你也会对如何改进图更有想法！另外，若你发现该篇 Mod 特别有用，记得给他们发个 [Kudosu](ZH:Kudosu "wikilink") 。
-　　是否能进入下一个环节，你的图获得了多少星星（优先度）是很重要的。如果摸图者喜欢你的图，他们或许会花费一个 Kudosu 以换给你的图一颗星星──你愈努力让你的图更好，这就愈有可能发生。你也可以透过摸其他人的图，并花费得来的 Kudosu 给你自己的图增加星星。如果你不擅长摸图，不怕，练习造就大触！就算是一些小小的建议亦能给其他作图者帮上不少忙，所以别害怕摸图。
+让自产图被 [Rank](/wiki/Beatmaps) 的第一步，也是最重要的一步，便是找人为你检查一下图。这个过程称为 [**Mod**](/wiki/Modding)，中文 osu! 圈中亦有人将其叫做**摸图**。你可以在 \#modreqs 频道、Working In Progress/Help（制作中的图／作图杂症板块）底下的子板块 Modding Queues（摸图串），或是在确认好对方的签名档、用户页面中所写的摸图规则之后直接询问对方等方式来找人帮你看看图。记住：所有 Mod 都有其用处，因为任何来自其他人的摸后感与想法都能为你提供更多观点，让你能将图做得更好。请务必考虑过任何来自于摸图者的建议，若你否决了其中一项，别忘了在图的讨论串底下发一篇详尽的回应清楚说明原因。这会让他们的摸图技巧进步，同时你也会对如何改进图更有想法！另外，若你发现该篇 Mod 特别有用，记得给他们发个 [Kudosu](/wiki/Glossary) 。
```

Comment:
it's a custom to leave 2 space character before each line in Chinese.......

I forgot to add this to zh-ASG.....
will add to #196